src/views/Home/components/MarketData.tsx

```diff
@@ -1,72 +1,176 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
 import numeral from 'numeral'
 import { NavLink } from 'react-router-dom'
-import { Card, CardContent, Spacer } from 'react-neu'
+import { Button, Card, CardContent, Spacer } from 'react-neu'
 
 import FancyValue from 'components/FancyValue'
 import Split from 'components/Split'
 import SimplePriceChart from 'components/SimplePriceChart'
 
-import useDpiTokenMarketData from 'hooks/useDpiTokenMarketData'
+import { productTokensBySymbol } from 'constants/productTokens'
+import useAllProductsMarketData from 'hooks/useAllProductsMarketData'
+import { PriceChartRangeOption } from 'constants/priceChartEnums'
 
 const MarketData: React.FC = () => {
-  const {
-    latestVolume,
-    latestMarketCap,
-    latestPrice,
-    prices,
-    hourlyPrices,
-  } = useDpiTokenMarketData()
-  const priceAtEpochStart = prices?.slice(-30)[0]?.[1] || 1
-  const epochPriceChange = (latestPrice || 0) - priceAtEpochStart
-  const dpiTokenIcon = {
-    src: 'https://index-dao.s3.amazonaws.com/defi_pulse_index_set.svg',
-    alt: 'DefiPulse Index Logo',
+  const allMarketData = useAllProductsMarketData()
+
+  const [indexSelector, setIndexSelector] = useState<number>(0)
+  const [indexMetaData, setIndeexMetaData] = useState<{
```

Comment:
index can be confusing since it also can refer to an array index... I've been guilty of it in the past but we should make an effort to move towards less ambiguous language.

Something like `productMetaData`/`setProductMetaData` seems good here.

Reply:
Great to know for the Big Refactor story that's upcoming